### DISCUSSION TITLE
Add points to progression widget on Completion Progress page

### DISCUSSION THREAD

**Discussion by UnliquidSnake** [RetroAchievements/RAWeb#2048] [Ideas]
Same thing as the change on Last Played

**Comment by wescopeland** [comment]
My biggest concern is the battle for space in these game list item components. The rows can start to become a bit cramped. The next ask will be for another stat like RetroPoints.

I'd like to see this handled a bit differently on the Completion Progress page. The direction I'd ultimately like to go is:

* Have the rows expand similar to what is on the user profile (we will need to lazy load the achievement list on expand)
* In addition to showing the achievement badges on expansion, have a small stats area which could contain this and other numbers.

**Comment by UnliquidSnake** [comment]
Actually, that sounds fine, just making it expandable like the Last Played sheets but with minor changes. The lines that are currently there I imagine would stay where they are (because they're all sorting indicators and it's kind of the main bulk of the column), and then in the expansion you could have the points. Or maybe remove the Beaten/Mastered line and put it as a hover-over message for the icon for beating/mastering a game. This could go a bunch of ways.
